editor/src/com/talosvfx/talos/editor/addons/scene/logic/components/SpriteRendererComponent.java

```diff
@@ -15,18 +15,20 @@
 import com.talosvfx.talos.editor.addons.scene.assets.GameAsset;
 import com.talosvfx.talos.editor.addons.scene.assets.GameAssetType;
 import com.talosvfx.talos.editor.addons.scene.logic.GameObject;
+import com.talosvfx.talos.editor.addons.scene.logic.IColorHolder;
 import com.talosvfx.talos.editor.addons.scene.widgets.property.AssetSelectWidget;
 import com.talosvfx.talos.editor.nodes.widgets.ValueWidget;
 import com.talosvfx.talos.editor.widgets.propertyWidgets.*;
 
 import java.util.function.Supplier;
 
-public class SpriteRendererComponent extends RendererComponent implements GameResourceOwner<Texture>, ISizableComponent {
+public class SpriteRendererComponent extends RendererComponent implements GameResourceOwner<Texture>, ISizableComponent, IColorHolder {
 
     public transient GameAsset<Texture> defaultGameAsset;
     public GameAsset<Texture> gameAsset;
 
     public Color color = new Color(Color.WHITE);
+    public Color worldColor = new Color();
```

Comment:
World colour isn't the best name, it's logical for the position but not for colour. 
Maybe the final colour is a better one?
Also mark this field as transient